example/example.js

```diff
@@ -0,0 +1,23 @@
+const { AudioProcessor } = require('../index'); // Assuming your index.js is in the parent directory
+
+async function runExample() {
+  const audioFilePath = '../audio/a.mp3';
+  const outputFilePath = './redacted_audio.wav';
```

Comment:
I think this generated file should be in a subfolder and should not be checked in (ie subfolder should appear in .gitiginore)